### PR TITLE
DOC: Tiny changes

### PR DESCRIPTION
**Pull request by j-t-1**

### CODE REVIEW

File: docs/dev/documentation.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ The title of the PR will be used as the first line of that combined commit messa
 
 The first comment within the commit will be used as the message body.
 
-See [dev intro](intro.html#commit-messages)  for more details.
+See [developer intro](intro.html#commit-messages)  for more details.
```
Comment (stefan6419846) [inline docs/dev/documentation.md:56]:
```suggestion
See [developer intro](intro.html#commit-messages) for more details.
```

File: docs/dev/documentation.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ def example(param1: int, param2: str) -> bool:
     """
 ```
 
-* The order of sections is (1) Args (2) Returns (3) Raises (4) Examples
+* The order of sections is 1. Args 2. Returns 3. Raises 4. Examples
```
Comment (stefan6419846) [inline docs/dev/documentation.md:35]: Which benefit does this have? I consider the old variant more readable.

Comment (j-t-1) [reply docs/dev/documentation.md]: How about a compromise?

> * The order of sections is 1) Args 2) Returns 3) Raises 4) Examples

Comment (stefan6419846) [reply docs/dev/documentation.md]: Does this really improve readability? I consider the original variant the most readable.

Comment (j-t-1) [reply docs/dev/documentation.md]: Reverted.

**Review by stefan6419846 — APPROVED** [review]

**Pull request merged**